docs/source/nlp/text_normalization.rst

```diff
@@ -0,0 +1,146 @@
+.. _text_normalization:
+
+Text Normalization Models
+==========================
```

Comment:
could you please add references at the end to google paper with tagger and dataset paper 'text norm challenge'?

Reply:
Yes. I added those.